lib/writer/block-write-stream.js

```diff
@@ -95,6 +98,30 @@ class BlockWriteStream extends stream.Writable {
     this.open()
   }
 
+  /**
+   * @summary Determine whether an error is considered a
+   * transient occurrence, and the operation should be retried
+   * Errors considered potentially temporary are:
+   *   - Mac OS: ENXIO, EBUSY
+   *   - Windows: ENOENT, UNKNOWN
+   *   - Linux: EIO
+   * @private
+   * @param {Error} error - Error
+   * @returns {Boolean}
+   * @example
+   * BlockWriteStream.isTransientError(error)
+   */
+  static isTransientError (error) {
```

Comment:
This function is a copy-paste of the `isTransientError` function in `BlockReadStream`. Can we move it somewhere else so that both streams can re-use the same function?

Reply:
I'd rather keep the streams self-contained, but can do – where'd you put it though?

Reply:
In some sort of `utils.js` file?

Reply:
Done